src/types/index.ts

```diff
@@ -330,6 +330,11 @@ export interface ExtrinsicData {
   extrinsicHash: string;
 }
 
+export interface ExtrinsicDataWithFees extends ExtrinsicData {
```

Comment:
Declaring the type here is OK, 

ideally we'd be able to provide the fees for all Extrinsic, but it seems like `Account.getTransactionHistory` populates `ExtrinsicData` from middleware, which doesn't have it.

It would be nice if SubQuery would have these fees, and then we could get away with only `ExtrinsicData`

Reply:
> ideally we'd be able to provide the fees for all Extrinsic, but it seems like Account.getTransactionHistory populates ExtrinsicData from middleware, which doesn't have it.

Exactly my point. Lets see what @monitz87 has to say here

Reply:
Can't they be included in subquery?

For now, I think extending `getTransactionHistory` to fetch fees makes sense so that we can return the same interface from both methods. For that one, you can use `entriesAt`, which returns entries for a query at a specific block

Reply:
If we go this route, I guess `getProtocolFees` should be extended to fetch for a set of tags using `entriesAt`. You would get all existing protocol fees, which shouldn't be too many, and filter them using the passed tags (any tags not present are considered to have 0 fees)